run/loader.go

```diff
@@ -1,16 +1,36 @@
 package run
 
 import (
+	"github.com/prometheus/client_golang/prometheus"
 	"github.com/relex/gotils/channels"
 	"github.com/relex/gotils/logger"
+	"github.com/relex/gotils/promexporter/promreg"
 	"github.com/relex/slog-agent/base"
 	"github.com/relex/slog-agent/base/bconfig"
+	"github.com/relex/slog-agent/defs"
 )
 
 // LoaderIface defines abstract configuration loader
 type LoaderIface interface {
```

Comment:
What is the point of this interface? Is it just for documentation purposes? I don't see any distinct "implementations" of it. I admit my understanding of Go interfaces is vague..